docs/evm/Build/Guides/gas-fee-on-flow-evm.md

```diff
@@ -0,0 +1,82 @@
+# **How gas fees work on Flow EVM?**
+
+### ***Revisiting Transaction Fee on Flow***
+
+Transaction Fee on Flow is broken down into [three components](https://developers.flow.com/learn/concepts/variable-transaction-fees) - ‘*Inclusion Fee*’ that accounts for the resources required to process a transaction due to the transaction’s core properties (byte size, number of signatures); ‘Execution Fee’ that accounts for the operational cost of running the transaction script, processing the results, sending results for verification, generating verification receipt, etc; and a *Surge* factor as a multiplicative factor to dynamically account for network pressure and market conditions.
+
+***Transaction fee = {inclusion fee + (execution effort * unit cost)} x surge***
```

Comment:
Line 7 is being flagged as an [error](https://github.com/onflow/docs/actions/runs/8010878669/job/21882877890?pr=588#step:7:59) by Vercel

Reply:
https://github.com/micromark/micromark-extension-mdx-expression/tree/main/packages/micromark-extension-mdx-expression#could-not-parse-expression-with-acorn

Reply:
I think the open braces `{` is causing it to evaluate the text as an expression

Reply:
Changed, thank you!

Reply:
🤞